core/src/View/Kokkos_ViewTraits.hpp

```diff
@@ -556,11 +556,21 @@ struct ViewTraits {
                       decltype(customize_view_arguments(
                           Impl::ViewArguments<value_type, array_layout,
                                               device_type, memory_traits>()))>;
-
-  enum { is_hostspace = std::is_same_v<MemorySpace, HostSpace> };
-  enum { is_managed = MemoryTraits::is_unmanaged == 0 };
-  enum { is_random_access = MemoryTraits::is_random_access == 1 };
-
+#ifdef KOKKOS_ENABLE_DEPRECATED_CODE_4
```

Comment:
Let's discuss whether this should be DEPRECATED_CODE_5 at the developer meeting today